test/unit/math/mix/fun/svd_V_test.cpp

```diff
@@ -0,0 +1,30 @@
+#include <test/unit/math/test_ad.hpp>
+
+TEST(MathMixMatFun, svd_V) {
```

Comment:
Same as the svd_U tests. Uncomment the tests one by one and see if you can find the problem